youtube-client/src/app/auth/pages/auth/auth.component.html

```diff
@@ -0,0 +1,23 @@
+<app-header></app-header>
```

Comment:
should be places in `AppComponent` since it's a header. You shouldn't include headers into pages, unless you need specific header in different cases